ansible/roles/automation/openclose_pr/tasks/main.yml

```diff
@@ -67,18 +67,19 @@
   ignore_errors: True
 
 - cron:
-    name: "openclose_{{ identifier }}"
-    minute: "0"
-    hour: "23"
-    weekday: "1,3,5"
+    name: "openclose_{{ item.name }}"
+    minute: "{{ item.minute }}"
+    hour: "{{ item.hour }}"
+    weekday: "{{ item.weekdays }}"
     job: >
            python3 {{ basedir }}/open_close_pr.py
            --config {{ basedir }}/config.yml
-           --id {{ identifier }}
+           --id {{ item.name }}
            --repo_path {{ basedir }}/freeipa
            --branch {{ branch }}
            --prci_config {{ prci_config }}
            --pr_against_upstream {{ pr_against_upstream | bool }}
            open_pr
     user: root
     state: present
+  with_items: "{{ nightly_jobs }}"
```

Comment:
missing spaces?

Reply:
You mean identation ? with_items should stay at the same level of sub-task

Reply:
yes, you right! sorry. Let's merge this PR.